cmd/argocd/commands/completion.go

```diff
@@ -203,8 +203,9 @@ To access completions in your current shell, run
 $ source <(argocd completion bash)
 Alternatively, write it to a file and source in .bash_profile
 
-For zsh, output to a file in a directory referenced by the $fpath shell
-variable.
+For zsh, add the following to your ~/.zshrc file:
+source <(argocd completion zsh)
+compdef _argocd argocd
```

Comment:
@dimisjim I might be doing something silly, but I'm getting this error:

```
% zsh
/Users/mcrenshaw/.zshrc:3: command not found: compdef
```

Reply:
are you running oh my zsh?
maybe it's a bit different on mac? (I am running it on linux)

have you tried adding to your ~/.zshrc file the following:

```
autoload -Uz compinit
compinit
```

as suggested [here](https://unix.stackexchange.com/a/477909/306647)?

Reply:
That works! So the complete solution for my Mac is

```
autoload -Uz compinit
compinit
source <(argocd completion zsh)
compdef _argocd argocd
```

Do you think it's worth adding that to the docs, or think it's enough a niche issue to justify leaving it out? Folks can always search the error message.

Reply:
I think it would be nice to add it to the docs.

Reply:
Sounds good! If you update the PR, I'll happily merge.  :-)

Reply:
sure, would this suffice? https://github.com/argoproj/argo-cd/pull/10535/commits/b87e6e4e7433545064f1b19a56819058c66d08bb

Reply:
Yep, that works! 

argocd_completion is generated from completion.go. You'll have to add the text in completion.go and then run `make clidocsgen-local`.

Reply:
Sure, ran `make clidocsgen`, made sure commits are signed, and I guess it should be good now.